File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/car/NewCarRequestDto.java
package ps.com.viajeros.dtos.car;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class NewCarRequestDto {
    private Long userId;
    private String brand;
    private String model;
    private String patent;
    private String color;
    private String fuel;
    private String kml;
    private boolean gnc;

}

File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Vehicle/NewCarRequestDto.ts
export class NewCarRequestDto {
    userId: number;
    brand: string;
    model: string;
    patent: string;
    color: string;
    fuel: string;    // Añadir este campo
    kml: string;
    gnc: boolean;
  
    constructor(
      userId: number,
      brand: string,
      model: string,
      patent: string,
      color: string,
      fuel: string,   // Añadir este campo en el constructor
      kml: string,
      gnc: boolean
    ) {
      this.userId = userId;
      this.brand = brand;
      this.model = model;
      this.patent = patent;
      this.color = color;
      this.fuel = fuel;  // Inicializar este campo
      this.kml = kml;
      this.gnc = gnc;
    }
  }
  